src/App.js

```diff
@@ -32,26 +36,84 @@ class App extends React.Component {
     try {
 
       let url = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_LOCATIONIQ_ACCESS_TOKEN}&q=${this.state.searchQuery}&format=json`
-      
+
       let cityResults = await axios.get(url)
-      console.log(cityResults.data)
+      // console.log(cityResults.data)
       this.setState({
         cityData: cityResults.data[0],
         showMapAndCityInfo: true
       })
-      } catch (error){
-          console.log(`Error Occured: ${error.response.status}, City Not Found`);
-        this.setState({
-          renderError: true,
-          errorMessage: `Error Occured: ${error.response.status}, City Not Found`
-        })
-    } 
+
+      // let serverUrl = `${process.env.REACT_APP_SERVER_URL}/weather?searchQuery=${this.state.searchQuery}`;
+      // let weatherResults = await axios.get(serverUrl);
+
+      // console.log(serverUrl);
+      // console.log(weatherResults.data);
+      // this.setState({
+      //   weatherData: weatherResults.data,
+      //   showWeatherData: true,
+      // })
+
+
+    } catch (error) {
+      console.log(`Error Occurred: ${error.response.status}, City Not Found`);
+      this.setState({
+        renderError: true,
+        errorMessage: `Error Occurred: ${error.response.status}, City Not Found`
+      })
+    }
+    this.getWeatherInfo();
+  }
+
+
+  getWeatherInfo = async () => {
+
+    try {
+      let serverUrl = `${process.env.REACT_APP_SERVER_URL}/weather?lat=${this.state.cityData.lat}&lon=${this.state.cityData.lon}`;
+      let weatherResults = await axios.get(serverUrl);
+
+      console.log(serverUrl);
+      console.log(weatherResults.data);
+      this.setState({
+        weatherData: weatherResults.data,
+        showWeatherData: true,
+      })
+
+
+    } catch (error) {
+      console.log(`Error Occurred: ${error.response.status}, City Not Found`);
+      this.setState({
+        renderErrorTwo: true,
+        errorMessageTwo: `Error Occurred: error status 500`
+      })
+    }
+
   }
 
 
 
+
+
+
+
+
+
+
   render() {
-    console.log(this.state.searchQuery);
+    // console.log(this.state);
+    // console.log(this.state.weatherData.data);
+
+    let weatherToRender = this.state.weatherData.map((weather, idx) => (
```

Comment:
- Good job so far on the updates! I see you have created your Weather.js component but need to now render the weather day there instead of App.js. 👍